variety/VarietyWindow.py

```diff
@@ -1684,30 +1680,21 @@ def move_to_trash(self, widget=None, file=None):
                 self.thumbs_manager.remove_image(file)
 
                 def _go():
-                    command = ''
-                    if find_executable('gio'):
-                        command = ['gio', 'trash', file.encode('utf-8')]
-                    elif find_executable('trash-put'):
-                        command = ['trash-put', file.encode('utf-8')]
-                    elif find_executable('kfmclient'):
-                        command = ['kfmclient', 'move', file.encode('utf-8'), 'trash:/']
-
-                    logger.info("Running trash command %s", command)
-                    if command:
-                        result = subprocess.call(command)
-                        if result != 0:
-                            logger.error("Trash resulted in error code %d", result)
-                            self.show_notification(
-                                _("Cannot delete"),
-                                _("Deleting to trash failed, check variety.log for more information."))
-                    else:
-                        logger.error("Delete to trash failed as no suitable program was found.")
+                    try:
+                        from gi.repository import Gio
```

Comment:
Gio is already imported above, we shouldn't need to add it again: https://github.com/varietywalls/variety/blob/master/variety/VarietyWindow.py#L28